CHANGELOG.rst

```diff
@@ -18,13 +18,20 @@ number of the code change for that issue.  These PRs can be viewed at:
     https://github.com/spacetelescope/drizzlepac/pulls
 
 
-3.7.1 (unreleased)
+3.7.2 (unreleased)
 ==================
-- Reverted #1798 until further testing is done with Photutils.
+
+- Added python 3.12 to testing matrix for Jenkins and github actions. [#1843]
 
 - ``manageInputCopies`` now copies successfully even if the original files were
   defined by full paths rather than being in the current working directory. [#1835]
 
+
+3.7.1 (unreleased)
```

Comment:
Should 3.7.1 (unreleased anyway) be rolled up into 3.7.2? Or should 3.7.2 above be changed to 3.7.1.